### PR TITLE
bump to Git 2.13.3 and git-lfs 2.2.1

### PR DESCRIPTION
**Pull request by shiftkey**
A few minor releases behind:

 - Git LFS - [v2.2.0](https://github.com/git-lfs/git-lfs/releases/tag/v2.2.0), [v2.2.1](https://github.com/git-lfs/git-lfs/releases/tag/v2.2.1)
 - Git - [v2.13.1](https://github.com/git/git/blob/master/Documentation/RelNotes/2.13.1.txt), [v2.13.2](https://github.com/git/git/blob/master/Documentation/RelNotes/2.13.2.txt), [v2.13.3](https://github.com/git/git/blob/master/Documentation/RelNotes/2.13.3.txt)
 - Git for Windows - [v2.13.1](https://github.com/git-for-windows/git/releases/tag/v2.13.1.windows.1), [v2.13.2](https://github.com/git-for-windows/git/releases/tag/v2.13.2.windows.1), [v2.13.3](https://github.com/git-for-windows/git/releases/tag/v2.13.3.windows.1)

- [x] tag and publish release

### CODE REVIEW

**Comment by shiftkey** [comment]
Putting this on hold until I can get to the bottom of whatever fresh hell Travis is in around mis-using the encrypted OAuth token :hurtrealbad:

**Comment by joshaber** [comment]
Providing the token through an env var seems to be working. I'm still not sure why the other methods didn't work 🤷‍♂️

**Comment by joshaber** [comment]
@shiftkey Once the Mac build completes, will this be good to merge?

File: .travis.yml [hunk 1]
```diff
@@ -37,8 +37,7 @@ branches:
 
 deploy:
   provider: releases
-  api_key:
-    secure: PZYs7hghA8bk+a9B9dsT4TcbmrGgPjVSKIXG+FGEX4ndd7thvYIuVNvGwY2qzw3QNPIaS8bsaR4u0OzO5PQ04maRcYBvkk3xWhB1VxMreokrz1lD0ee5UBzZcuKMuXpBatMJHQfbzRpJxD6c9ipP56lNeTB+ZZnKoGeTUO6BA2neDSHmTwU5Ps3aDRQ8VjK9/A6TzSQqczKBmM7t4g2RdC2+b0qH+2Waw2Rdsvf34SQiIixomXXE3sJaxGSXGtFg1+IOSnirJ/qT8sTcTwVRfCLzuQdbVfNbWPPRkBpKNOMxY8qJeEEa3TNu2yKstWva+JDLuDwQ/D5Lxl1YN7yFBSdiTODbfLRaunvbphrLMiho3QGcSYTqclNxrcjB5cuqq58jC6PR5bK/LGQd6jgUhRkuJOhcxpj5eL9hmho8Yr66laROYxCnPlCENfqlXxcpCuqwMDvwxoD8U9u6hViDU2FtMS/9QwNrgkKBOCiOmiWm29NZR1S9o68DZ3trNn63F/IVaZpkfQhLg+waLKoqL54xtOmhQwEzUyyTWF3jcuLLluCpWAj6Vcttj9zLFQ0sdUWEJiuqhpmOZcgicoFd+j5eNZz7Q9l3UoyAtOowuNLHCqK0IO1Cb9Ib6+L7PscUKAlgLVMDSHe1RxuJ4EhwraqTUrXUs+aq57qyEBn9AV8=
+  api_key: $GITHUB_TOKEN
```
Comment (shiftkey) [inline .travis.yml]: Ah yes, we've put this away in [repository settings](https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings). Good idea.

**Comment by shiftkey** [comment]
@joshaber this is good to go :shipit:

**Pull request merged**